Examples/BookingSystem.AspNetCore/Stores/SellerStore.cs

```diff
@@ -22,13 +23,13 @@ public AcmeSellerStore(bool useSingleSellerMode)
         }
 
         // If the Seller is not found, simply return null to generate the correct Open Booking error
-        protected override ILegalEntity GetSeller(SellerIdComponents sellerIdComponents)
+        protected override ValueTask<ILegalEntity> GetSellerAsync(SellerIdComponents sellerIdComponents)
         {
             // Note both examples are shown below to demonstrate options available. Only one block of the if statement below is required for an actual implementation.
             if (_useSingleSellerMode)
             {
                 // For Single Seller booking systems, no ID will be available from sellerIdComponents, and this data should instead come from your configuration table
-                return new Organization
+                return new ValueTask<ILegalEntity>(new Organization
```

Comment:
This shouldn't need to be a ValueTask as it's a static value, only the interface needs this and .NET should do some magic to figure out to return one

```suggestion
                return new Organization
```